docs/data-create.rst

```diff
@@ -1,73 +1,80 @@
   $ mkdir "$PWD"/dataset "$PWD"/feature "$PWD"/boundary "$PWD"/centroid
 
-2. Pull the HistomicsML-TA docker image.
+  Superpixel boundaries and centroids (x,y) coordinates are stored in the *boundary* and *centroid* folders. Features extracted from superpixels are stored in the *feature* folder. The *dataset* folder contains the final transformed data in .h5 format that is ready for ingestion (see below). Finally, an *svs* directory contains whole-slide image files. Data from a single slide is included in the Docker image as an example.
+
+2. Download the dataset creation Docker container
+====================================================================
+
+Download the HistomicsML dataset creation Docker container
 
 .. code-block:: bash
 
   $ docker pull cancerdatascience/hml_dataset_gpu:1.0
 
-3. Create boundaries and centroids of superpixels.
+Use ``SuperpixelSegmentation.py`` to generate superpixel boundaries and centroids
 
 .. code-block:: bash
 
   $ docker run -it --rm --name createboundary -v "$PWD"/boundary:/boundary -v "$PWD"/centroid:/centroid cancerdatascience/hml_dataset_gpu:1.0 python scripts/SuperpixelSegmentation.py --superpixelSize 64 --patchSize 128
 
-.. note:: Adjustable parameters of ``SuperpixelSegmentation.py`` and ``FeatureExtraction.py`` are following::
+.. note:: 
+  Parameters of the superpixel segmentation script ``SuperpixelSegmentation.py`` can be adjusted to change the size, shape, and thresholding of superpixels to discard background regions
+  
+  -superpixelSize        An approximate edge length of each superpixel. Range is [8, 256]. Default 64.
+  
+  -patchSize             Patch size of each superpixel. Range is [8, 512]. Default 128.
 
---superpixelSize        An approximate edge length of each superpixel.
-                        Range is [8, 256]. Default 64.
---patchSize             Patch size of each superpixel. Range is [8, 512]. Default 128.
+3. Extract features
+====================================================================
 
-4. Create features of superpixels.
+On a CPU system
 
 .. code-block:: bash
 
-  # use the command line below if using CPU.
   $ docker run -it --rm --name extractfeatures -v "$PWD"/feature:/feature -v "$PWD"/centroid:/centroid cancerdatascience/hml_dataset_gpu:1.0 python scripts/FeatureExtraction.py --superpixelSize 64 --patchSize 128
-  # use the command line below if using GPU. Current verion supports CUDA 9.0, Linux x86_64 Driver Version >= 384.81
+
+On a GPU system (currently supporting CUDA 9.0, Linux x86_64 Driver Version >= 384.81):
+
+.. code-block:: bash
+
   $ docker run --runtime=nvidia -it --rm --name extractfeatures -v "$PWD"/feature:/feature -v "$PWD"/centroid:/centroid cancerdatascience/hml_dataset_gpu:1.0 python scripts/FeatureExtraction.py --superpixelSize 64 --patchSize 128
 
-5. Create HistomicsML dataset.
+.. note:: 
+  The patch size utilized by ``FeatureExtraction.py`` can be adjusted:
```

Comment:
@slee172 Separate out the parameters of FeatureExtraction.py and SuperpixelSegmenation.py to avoid confusion.

Reply:
We may have to fix this after correcting the most current updates.

Reply:
The parameter information between FeatureExtraction.py and SuperpixelSegmenation.py are separated.